buildconfig/CMake/WindowsNSIS.cmake

```diff
@@ -7,14 +7,26 @@
 ###########################################################################
 set( CPACK_GENERATOR "NSIS" )
 set( CPACK_INSTALL_PREFIX "/")
-set( CPACK_NSIS_DISPLAY_NAME "Mantid${CPACK_PACKAGE_SUFFIX}")
 set( CPACK_PACKAGE_NAME "mantid${CPACK_PACKAGE_SUFFIX}" )
-set( CPACK_PACKAGE_INSTALL_DIRECTORY "MantidInstall${CPACK_PACKAGE_SUFFIX}")
 set( CPACK_PACKAGE_INSTALL_REGISTRY_KEY "${CPACK_PACKAGE_NAME}" )
 set( CPACK_NSIS_INSTALL_ROOT "C:")
-set( CPACK_PACKAGE_ICON "${CMAKE_CURRENT_SOURCE_DIR}/images\\\\MantidPlot_Icon_32offset.png" )
-set( CPACK_NSIS_MUI_ICON "${CMAKE_CURRENT_SOURCE_DIR}/images\\\\MantidPlot_Icon_32offset.ico" )
-set( CPACK_NSIS_MUI_UNIICON "${CMAKE_CURRENT_SOURCE_DIR}/images\\\\MantidPlot_Icon_32offset.ico" )
+set( CPACK_PACKAGE_ICON "${CMAKE_CURRENT_SOURCE_DIR}\\\\images\\\\MantidPlot_Icon_32offset.png" )
+set( CPACK_NSIS_MUI_ICON "${CMAKE_CURRENT_SOURCE_DIR}\\\\images\\\\MantidPlot_Icon_32offset.ico" )
+set( CPACK_NSIS_MUI_UNIICON "${CMAKE_CURRENT_SOURCE_DIR}\\\\images\\\\MantidPlot_Icon_32offset.ico" )
+
+# Choose the proper suffix for the build.
+if ( CPACK_PACKAGE_SUFFIX STREQUAL "nightly" )
+  set ( WINDOWS_CAPITALIZED_PACKAGE_SUFFIX "Nightly" )
+elseif ( CPACK_PACKAGE_SUFFIX STREQUAL "unstable" )
+  set ( WINDOWS_CAPITALIZED_PACKAGE_SUFFIX "Unstable" )
+else()
+  # this is the release suffix, which is empty
+  set ( WINDOWS_CAPITALIZED_PACKAGE_SUFFIX "" )
+endif()
+
+# have the properly capitalsed name for the start menu and install folder
+set( CPACK_NSIS_DISPLAY_NAME "Mantid${WINDOWS_CAPITALIZED_PACKAGE_SUFFIX}")
+set( CPACK_PACKAGE_INSTALL_DIRECTORY "MantidInstall${WINDOWS_CAPITALIZED_PACKAGE_SUFFIX}")
```

Comment:
How about `set( CPACK_PACKAGE_INSTALL_DIRECTORY "${CPACK_NSIS_DISPLAY_NAME}Install")` to the package name is kept together?